src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,58 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+import React from 'react';
+import { getInfoFromTab } from '../../functions/getInfoFromTab';
+import { Tab } from '../../types/Tab';
+
+type Props = {
+  tabs: Tab[],
+  selectedTabId: string,
+  onTabSelected: (tab: Tab) => void,
+};
+
+export const Tabs: React.FC<Props> = (props) => {
+  const {
+    tabs,
+    selectedTabId,
+    onTabSelected,
+  } = props;
+
+  const isActive = tabs.some(tab => tab.id === selectedTabId)
+    ? selectedTabId
+    : tabs[0].id;
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {
+            tabs.map(tab => (
+              <li
+                className={classNames({ 'is-active': isActive === tab.id })}
+                data-cy="Tab"
+                key={tab.id}
+              >
+                <a
+                  href={`#${tab.id}`}
+                  data-cy="TabLink"
+                  onClick={() => {
+                    if (selectedTabId !== tab.id) {
+                      onTabSelected(tab);
+                    }
+                  }}
```

Comment:
when callback just call other function - you can leave it in murkup
but when it contain some logic, it will be better to move it from markup to some variable
```suggestion
const handleTabClick = (id: number) => {
	if (selectedTabId !== tab.id) {
		onTabSelected(tab);
	}
}
	...	
  onClick={() => handleTabClick(tab.id)}
```